pages/bookmarks.jsx

```diff
@@ -37,7 +37,7 @@ export default React.createClass({
     return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
   },
   getAnonymousContent() {
-    return (<p>Are you Mozilla Staff? <button className="btn btn-link inline-link" onClick={this.handleSignInBtnClick}>Sign in</button>.</p>);
+    return (<p>Are you Mozilla Staff? <a href="javascript:void(0)" onClick={this.handleSignInBtnClick}>Sign in</a>.</p>);
```

Comment:
@Pomax this is the ticket this PR tries to fix: https://github.com/mozilla/network-pulse/issues/365 When I created that ticket I was thinking it *should* be an `<a>` since it ultimately redirects users to an external link.